src/UiCalendarWeek.vue

```diff
@@ -172,7 +172,7 @@ export default {
     }
 
     &.is-today {
-        font-weight: bold;
```

Comment:
Depending on the font, sometimes `bold` can be different from the 600 weight, usually it's 700.

So perhaps we can rename the existing 600 weight variable `$font-weight--semibold`, and have `$font-weight--bold` be 700, for use here and any other place we have `font-weight: bold`?

Reply:
Yes, I have been thinking about this. It indeed makes some sense, but I was just thinking whether it makes sense to just do it for only one case where bold is used (and whether we actually need this bold at all). But indeed there might be cases in the future where we might need it so it might become a breaking change.

If you think it's the case then I can implement this, no problem.